dist/I18N-Collate/t/I18N-Collate.t

```diff
@@ -7,7 +7,7 @@ use strict;
 no warnings;
 
 BEGIN {
-    require Config; import Config;
+    require Config; Config->import;
```

Comment:
This individual is a require, then an import in a BEGIN... does perl have some kind of short hand for that?

Reply:
You're not wrong, but I'd rather do that in a separate PR.

Reply:
See #23112.